app/common/model/src/main/java/io/syndesis/common/model/connection/ConfigurationProperty.java

```diff
@@ -37,15 +37,15 @@
      */
     String MULTIPLE_SEPARATOR = ",";
 
-    class Builder extends ImmutableConfigurationProperty.Builder {
+    class Builder extends io.syndesis.common.model.connection.ImmutableConfigurationProperty.Builder {
```

Comment:
nit

```suggestion
    class Builder extends ImmutableConfigurationProperty.Builder {
```

Reply:
For some reason my IDE forces that repeatedly. I've tried to remove but with no luck. I'll have a further look to have it cleaner, I don't like it either.

Reply:
Fixed